mudlet-mapper.xml

```diff
@@ -4817,7 +4817,7 @@ mmp.speedWalkPath = mmp.speedWalkPath or {}
 mmp.speedWalkDir = mmp.speedWalkDir or {}
 
 
-local newversion = &quot;developer&quot;
+local newversion = &quot;18.2.5&quot;
```

Comment:
Can you also undo this change? Our release script relies on the line staying the same in the repository here.